app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -42,9 +43,36 @@ def create
         end
       end
 
+      def update
+        super
```

Comment:
If I put `super` at the bottom then you get the flash notifications in this order:
```
Disabling Simple Content Access for Org B
Successfully updated Org B
```
I think that's confusing because it implies that the SCA toggle was already successful.